numba/tests/test_ipython_magic.py

```diff
@@ -0,0 +1,22 @@
+from unittest import TestCase
+from IPython.terminal.interactiveshell import TerminalInteractiveShell
+
+code = u"""\
+import numba
+
+@numba.njit
+def f(x):
```

Comment:
Note from discussion: we should also add a test where something defined outside the cell is referred to.